### ISSUE TITLE
News tone kickers fail WCAG over Review tone background

### ISSUE THREAD

**Issue by paperboyo** [guardian/frontend#8126]
Mornin’

2000 words worth; as seen on the Network Front today (almost):

![wcag-front-fail](https://cloud.githubusercontent.com/assets/6032869/5996833/d98bdd4c-aaa8-11e4-9bc2-6cb85e2124df.png)

![wcag results](https://cloud.githubusercontent.com/assets/6032869/5996834/dcb79178-aaa8-11e4-9c7b-714fa467e903.PNG)

Regards
Mateusz

**Comment by ScottPainterGNM** [comment]
This appears to have been changed to a lighter shade of blue now to avoid this, will close off issue.
